bedrock/styleguide/templates/styleguide/base.html

```diff
@@ -54,7 +53,7 @@
 
 <h1>{% block content_title %}{{ page_title }}{% endblock %}</h1>
 
-{% block subnav %}
+{% block styleguide_subnav %}
```

Comment:
I believe I changed it when it was referencing base-resp.html but I will double check. Without the name change, it added the prev/next menu items twice :)